tests/unit/confidant/models/credential_test.py

```diff
@@ -43,6 +43,28 @@ def test_not_equals(mocker):
     assert cred1.equals(cred2) is False
 
 
+def test_not_equals_different_tags(mocker):
```

Comment:
should we add a test to check for tags equal?

Reply:
Sure - I modified an existing test